### ISSUE TITLE
Video - Playback and Volume Control

### ISSUE THREAD

**Issue by jinky32** [ThreeUK/digital-library.three.co.uk#17]
**Requirement**: There needs to be a mechanism to stop or pause any video or audio content.

**Why is this required?** All users need to be able to control the playback on a webpage; in addition, screen reader users need to be able to control the playback of content on a page in relation to the screen reader output, to allow them to view the content in a timely fashion.

**Requirement**: There needs to be a mechanism to control the volume independently from the system level volume.

**Why is this required?** All users need to be able to control the audio output from a webpage; in addition, screen reader users need to be able to control the volume of audio content on a page in relation to the volume of the screen reader output, to allow them to hear either one clearly.

**Comment by papafed** [comment]
For reasons of practicality it's likely that all video will continue to be hosted and served by Youtube. Their player has all of these features, so me must ensure that our embed code enables them.

**Comment by jinky32** [comment]
And perhaps suggest to teams generating videos that they use something like TunestoTube http://www.tunestotube.com/ or follow guidelines at https://support.google.com/youtube/answer/2734799?hl=en for transcripts

**Comment by beasers** [comment]
We will soon be generating a lot of videos (in Franklin) for device support. My new plan is basically to show people how to do everything with 30 second clips. It'll definitely be hosted via YT. Harry - who makes the support videos - already includes captions and transcripts in the video description.
Franklin are also (I think) committing to contributing a reusable video asset to the library, so we will look into the accessibility best practice for video.
What I am slightly confused about is that this stop, pause, volume controls are being called out as separate requirements, since YT's player includes these natively. What am I missing?

**Comment by VinceButcher** [comment]
If we are using YT and they already have these functions then we don't need to do anything extra
